src/backend/machobj.c

```diff
@@ -2310,6 +2310,7 @@ void MachObj::addrel(int seg, targ_size_t offset, symbol *targsym,
         unsigned targseg, int rtype, int val)
 {
     Relocation rel;
+    memset(&rel, 0, sizeof(rel));
```

Comment:
All the fields of rel are assigned values here. Adding a memset() is completely redundant, and makes it slower.